src/test/java/DirectionsServletTest.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google LLC
```

Comment:
Any reason why this file is commented out? At first glance this looks functional

Reply:
Nope, it is not functional. Just wanted to get a first glance on the big change in structure of the dependency injection pattern. However, this is fixed now and the tests should work! Sorry for the confusion.

Reply:
`RESOLVED`